test/test_ops.py

```diff
@@ -0,0 +1,206 @@
+      SkipEntry('log2', skip=reference_eager_int_skip),
+      SkipEntry(
+          'log_softmax', variant_name='dtype', skip=reference_eager_all_skip),
+      SkipEntry('log', skip=reference_eager_int_skip),
+      SkipEntry('logsumexp', skip=reference_eager_float_skip),
+      SkipEntry(
+          'max',
+          variant_name='reduction_with_dim',
+          skip=reference_eager_all_skip),
+      SkipEntry('mean', skip=reference_eager_float_skip),
+      SkipEntry(
+          'min',
+          variant_name='reduction_with_dim',
+          skip=reference_eager_all_skip),
+      SkipEntry('norm', variant_name='inf', skip=reference_eager_all_skip),
+      SkipEntry('norm', variant_name='nuc', skip=reference_eager_all_skip),
+      SkipEntry('norm', skip=reference_eager_all_skip),
+      SkipEntry('pinverse', skip=reference_eager_float_skip),
+      SkipEntry('prod', skip=reference_eager_all_skip),
+      SkipEntry('reciprocal', skip=reference_eager_int_skip),
+      SkipEntry('repeat', skip=reference_eager_all_skip),
+      SkipEntry('rsqrt', skip=reference_eager_int_skip),
+      SkipEntry(
+          'rsub', variant_name='rsub_scalar', skip=reference_eager_all_skip),
+      SkipEntry('scatter_add', skip=reference_eager_all_skip),
+      SkipEntry('scatter', skip=reference_eager_all_skip),
+      SkipEntry('sigmoid', skip=reference_eager_int_skip),
+      SkipEntry('sin', skip=reference_eager_int_skip),
+      SkipEntry('sinh', skip=reference_eager_int_skip),
+      SkipEntry('sort', skip=reference_eager_all_skip),
+      SkipEntry('sqrt', skip=reference_eager_int_skip),
+      SkipEntry('squeeze', skip=reference_eager_all_skip),
+      SkipEntry('std_mean', skip=reference_eager_float_skip),
+      SkipEntry('std', skip=reference_eager_float_skip),
+      SkipEntry('sum', skip=reference_eager_all_skip),
+      SkipEntry('svd', skip=reference_eager_float_skip),
+      SkipEntry('symeig', skip=reference_eager_float_skip),
+      SkipEntry('t', skip=reference_eager_all_skip),
+      SkipEntry('tan', skip=reference_eager_int_skip),
+      SkipEntry('tanh', skip=reference_eager_int_skip),
+      SkipEntry('tile', skip=reference_eager_all_skip),
+      SkipEntry('to_sparse', skip=reference_eager_all_skip),
+      SkipEntry('topk', skip=reference_eager_all_skip),
+      SkipEntry('transpose', skip=reference_eager_all_skip),
+      SkipEntry('var', skip=reference_eager_float_skip),
+  }
+
+  for op in op_db:
+    for op_name, variant_name, skip in skips:
+      if op.name == op_name and op.variant_test_name == variant_name:
+        if op.skips == ():
+          op.skips = (skip,)
+        else:
+          op.skips += (skip,)
+
+
+# Add skips to OpInfo (if any) for this test suite.
+update_skips()
+
+
+class TestOpInfo(TestCase):
+
+  def compare_with_eager_reference(self, torch_fn: Callable,
+                                   sample_input: SampleInput, **kwargs) -> None:
+
+    def cpu(sample: SampleInput):
```

Comment:
I think this should be moved to `SampleInputs` (just like it has `numpy()`)

Reply:
More generically we could expose a "tensor_map" operation which would let people apply arbitrary lambdas to tensors in the sample input, and then add a ".cpu()" sugar like ".numpy()". This is a cool idea and we would accept this PR. Would you file an issue in the PyTorch Github repo so we can track it, @kshitij12345?